piccolo/engine/sqlite.py

```diff
@@ -418,28 +419,49 @@ async def get_connection(self) -> Connection:
 
     ###########################################################################
 
+    async def _get_inserted_id(
+        self, cursor, tablename: t.Optional[str]
+    ) -> t.Any:
+        """
+        If `id` column is non-integer
+        `ROWID` and `id` will return different types.
+        Need to query by `lastrowid` to get `id`s in SQLite prior to 3.35.0.
+        """
+        # TODO: Add RETURNING clause for > 3.35.0
+        if version.parse(sqlite3.sqlite_version) < version.parse("3.35.0"):
+            await cursor.execute(
+                f"SELECT id FROM {tablename} WHERE ROWID = {cursor.lastrowid}"
+            )
+            response = await cursor.fetchone()
+            return response["id"]
```

Comment:
We'll have to be careful here. There are two options to prevent any Python 3.9 breakage when merging:

1. Remove the `if` statement
2. Add the RETURNING clause

Reply:
I agree with removing the `if` and keeping the comment for future referrences.